VRDR.CLI/Program.cs

```diff
@@ -40,32 +40,32 @@ static int Main(string[] args)
                 certificationRole.Add("display", "Physician");
                 deathRecord.CertificationRole = certificationRole;
 
-                // InterestedPartyIdentifier
-                var ipId = new Dictionary<string, string>();
-                ipId["system"] = "http://hl7.org/fhir/sid/us-npi";
-                ipId["value"] = "0000000000";
-                deathRecord.InterestedPartyIdentifier = ipId;
-
-                // InterestedPartyName
-                deathRecord.InterestedPartyName = "Example Hospital";
-
-                // InterestedPartyAddress
-                Dictionary<string, string> address = new Dictionary<string, string>();
-                address.Add("addressLine1", "10 Example Street");
-                address.Add("addressLine2", "Line 2");
-                address.Add("addressCity", "Bedford");
-                address.Add("addressCounty", "Middlesex");
-                address.Add("addressState", "MA");
-                address.Add("addressZip", "01730");
-                address.Add("addressCountry", "US");
-                deathRecord.InterestedPartyAddress = address;
-
-                // InterestedPartyType
-                Dictionary<string, string> type = new Dictionary<string, string>();
-                type.Add("code", "prov");
-                type.Add("system", "http://terminology.hl7.org/CodeSystem/organization-type");
-                type.Add("display", "Healthcare Provider");
-                deathRecord.InterestedPartyType = type;
+                // // InterestedPartyIdentifier
+                // var ipId = new Dictionary<string, string>();
+                // ipId["system"] = "http://hl7.org/fhir/sid/us-npi";
+                // ipId["value"] = "0000000000";
+                // deathRecord.InterestedPartyIdentifier = ipId;
+
+                // // InterestedPartyName
+                // deathRecord.InterestedPartyName = "Example Hospital";
+
+                // // InterestedPartyAddress
+                // Dictionary<string, string> address = new Dictionary<string, string>();
+                // address.Add("addressLine1", "10 Example Street");
+                // address.Add("addressLine2", "Line 2");
+                // address.Add("addressCity", "Bedford");
+                // address.Add("addressCounty", "Middlesex");
+                // address.Add("addressState", "MA");
+                // address.Add("addressZip", "01730");
+                // address.Add("addressCountry", "US");
+                // deathRecord.InterestedPartyAddress = address;
+
+                // // InterestedPartyType
+                // Dictionary<string, string> type = new Dictionary<string, string>();
+                // type.Add("code", "prov");
+                // type.Add("system", "http://terminology.hl7.org/CodeSystem/organization-type");
+                // type.Add("display", "Healthcare Provider");
+                // deathRecord.InterestedPartyType = type;

```

Comment:
once this branch is rebased, these tests should be uncommented out

Reply:
INterested part is history.  Not coming back.

Reply:
InterestedParty that is.